lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac.rb

```diff
@@ -0,0 +1,34 @@
+module MiqAeMethodService
+  module MiqAeServiceRbac
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def find_ar_object_by_id(id)
+        Rbac.search(:class => model, :targets => [id],
+                    :results_format => :objects).try(:first).try(:first)
+      end
+
+      def all
+        wrap_results(Rbac.search(:class => model, :results_format => :objects).try(:first))
+      end
+
+      def count
+        Rbac.search(:class => model).try(:first).count
+      end
+
+      def first
+        all.first
+      end
+
+      def filter_objects(objs)
+        return objs if objs.nil?
+        array = Array.wrap(objs)
+        return [] if array.empty?
```

Comment:
take that back, rbac will return an empty array if an empty array is passed in.
So just skip the blank / empty check here.